src/test/assembly/asm/nvptx-types.rs

```diff
@@ -1,6 +1,7 @@
 // no-system-llvm
 // assembly-output: emit-asm
 // compile-flags: --target --nvptx64-nvidia-cuda
+// compile-flags: -Z merge-functions=disabled
 // only-nvptx64
 // ignore-nvptx64
```

Comment:
Remove the `only-nvptx64` and `ignore-nvptx64` flags. We should always run these tests.

Reply:
done